lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +23,51 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class AppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    IOS = "ios"
+    ANDROID = "android"
+    PLATFORM = (
+        (IOS, "iOS"),
+        (ANDROID, "Android"),
+    )
+    KEY_FIELDS = ('platform', 'version')  # combination of mobile platform and version is unique
+    platform = CharField(max_length=50, choices=PLATFORM, blank=False)
+    version = CharField(max_length=50, blank=False)
+    major_version = IntegerField()
+    minor_version = IntegerField()
+    patch_version = IntegerField()
+    expire_at = DateTimeField(null=True, blank=True, verbose_name="Last Supported Date")
+
+    class Meta:
+        ordering = ['major_version', 'minor_version', 'patch_version']
+
+    @classmethod
+    def latest_version(cls, platform):
+        """ returns latest supported version for a platform """
+        active_configs = cls.objects.current_set().filter(platform=platform, enabled=True).reverse()
```

Comment:
You can add a `.first()` to the end to just fetch the top item from the database.

https://docs.djangoproject.com/en/1.8/ref/models/querysets/#first